[WIP] Sprint: Add/update alt-text for images (#222)
nebari-dev/nebari-docs: docs/docs/troubleshooting.mdx

> Once all environments are migrated, you will be able to delete the `default` and `filesystem` namespaces using the `delete` option from the conda-store UI:

Reviewer: ```suggestion
Once all environments are migrated, you will be able to delete the `default` and `filesystem` namespaces by clicking on their respective delete buttons in the conda-store UI:
```

> ![delete-namespace ](/img/troubleshooting/delete_namespace.png)

Reviewer: I'll work on this
